js/pages/samples.js

```diff
@@ -390,7 +390,7 @@ const henry_ford = {
         [
             'Henry Ford Medical Center - Livonia',
             '29200 Schoolcraft Road<br>Radiology, Floor 1<br>Livonia, MI 48150<br>(734) 523-1050',
-            'Mondays, Tuesdays and Wednesdays 9:00 a.m. - 4:00 p.m.',
+            'Mondays, Tuesdays and Wednesdays 9:30 a.m. - 3:30 p.m.<br> Every 2<sup>nd</sup> and 4<sup>th</sup> Saturday  of the month, 8:00 a.m. to 11:30 a.m.',
```

Comment:
Looks like the documentation describes `Mondays, Tuesdays, and Wednesdays 9:30 a.m. to 3:30 p.m.` ( `to` isntead of  `-`). Please help check it.